### ISSUE TITLE
Question about tabstop

### ISSUE THREAD

**Issue by pablox-cl** [tpope/vim-sleuth#19]
I have the following problem, I normally use spaces instead of tabs, but some projects still use them. It looks like the vim-sleuth could help me to ease the setup. After installing it, I removed the `tabstop`, `softtabstop`, `expandtab` and `shiftwidth` from my `vimrc`.

Problem is while it seems to work quite fine on my own files, and new ones from my own projects, it sets `tabstop` to `8` (which I believe is vim default). I normally work using a laptop with a tiling window manager so the "width" of the window for me is somewhat critical. And, for example open json tabbed files, results in something hard to wield.

Setting `tabstop` to `4`, it seems to improve the file a lot, but I'm not sure if this could interfer with the plugin itself (vim documentation states having set to something different to 8 it could led to issues while printing, but honestly, I have never used vim for printing, so it's a non-issue for me).

So, tl; dr → is it acceptable to the plugin, only include `set tabstop=4` from the `vimrc` while removing, `expandtab`, `shiftwidth`, `expandtab` and `softtabstop`? (I still have `set smarttab` on, and seems ok...)
